core/src/main/java/com/linecorp/armeria/common/HeadersSanitizer.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+import java.util.function.BiFunction;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * A sanitizer that sanitizes {@link HttpHeaders}.
+ */
+public interface HeadersSanitizer<T> extends BiFunction<RequestContext, HttpHeaders, T> {
+    /**
+     * Returns the default text {@link HeadersSanitizer}.
+     */
+    static HeadersSanitizer<String> ofText() {
+        return TextHeadersSanitizer.INSTANCE;
```

Comment:
Should we apply this default `TextHeadersSanitizer` to
https://github.com/line/armeria/blob/a52c45edd85a2d8a157a19761d2b6b7d191beea8/core/src/main/java/com/linecorp/armeria/common/logging/TextLogFormatterBuilder.java#L106-L107

Reply:
Fixed ! 👍